### PR TITLE
Create C03_12

### PR DESCRIPTION
**Pull request by DeedeeDL**
The program will create a  pattern like a right angle triangle with the number increased by 1.

### CODE REVIEW

File: src/main/java/org/example/C03_12__PatternAngledTriangleIncreasingNumber.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_12__PatternAngledTriangleIncreasingNumber {
```
Comment (andreiursudev) [inline src/main/java/org/example/C03_12__PatternAngledTriangleIncreasingNumber.java]: "AngledTriangle" does not make much sense. Better "RightAngledTriangle"

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C03_12__PatternAngledTriangleIncreasingNumber.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C03_12__PatternAngledTriangleIncreasingNumber.java]: done

File: src/main/java/org/example/C03_12_RightAngledTriangle.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import java.util.Scanner;
 
-public class C03_12__PatternAngledTriangleIncreasingNumber {
+public class C03_12_RightAngledTriangle {
```
Comment (andreiursudev) [inline src/main/java/org/example/C03_12_RightAngledTriangle.java]: Name does not describe the problem.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C03_12_RightAngledTriangle.java (hunk 2 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C03_12_RightAngledTriangle.java]: now ?:D

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**